### PR TITLE
able to send emails

### LINKED ISSUE #182: send emails according to events

### PR DESCRIPTION
**Pull request by amneet270**
user is able to send emails 
closes #182

### CODE REVIEW

File: src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java [hunk 1]
```diff
@@ -30,13 +34,10 @@
...
     public void onApplicationEvent(ApplicationReadyEvent event) {
         log.info("Running post start up configuration...");
-        Flux.just("demo", "jeff", "test")
-                .filterWhen(this::filterPreExistingUsers)
-                .map(this::buildDemoUser)
-                .collectList()
-                .subscribe(users -> godzillaUserRepository.saveAll(users)
-                    .doOnError(log::error)
-                    .subscribe(this::logDemoUserCreation));
+        sendEmailService.sendEmail("amneet.s.270@gmail.com", "test", "this is a test").subscribe();
```
Comment (obonobo) [inline src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java:37]: Hey can you just reformat this section to how it was before:

```java
        Flux.just("demo", "jeff", "test")
                .filterWhen(this::filterPreExistingUsers)
                .map(this::buildDemoUser)
                .collectList()
                .subscribe(users -> godzillaUserRepository.saveAll(users)
                    .doOnError(log::error)
                    .subscribe(this::logDemoUserCreation));
```

Hard to read with this auto formatting

File: src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java [hunk 2]
```diff
@@ -46,18 +47,12 @@ public void onApplicationEvent(ApplicationReadyEvent event) {
      * @return
      */
     private Mono<Boolean> filterPreExistingUsers(String username) {
-        return godzillaUserRepository
-                .findByUsername(username)
-                .map(u -> false)
-                .defaultIfEmpty(true);
+        return godzillaUserRepository.findByUsername(username).map(u -> false).defaultIfEmpty(true);
```
Comment (obonobo) [inline src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java:50]: Same here (reformat to how it was before):

```java
        return godzillaUserRepository
                .findByUsername(username)
                .map(u -> false)
                .defaultIfEmpty(true);
```

File: src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java [hunk 3]
```diff
@@ -46,18 +47,12 @@ public void onApplicationEvent(ApplicationReadyEvent event) {
...
     }
 
     private GodzillaUser buildDemoUser(String username) {
-        return GodzillaUser.builder()
-                .username(username)
-                .password(passwordEncoder.encode("demo"))
-                .authorities("ROLE_USER")
-                .build();
+        return GodzillaUser.builder().username(username).password(passwordEncoder.encode("demo"))
```
Comment (obonobo) [inline src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java:54]: And here (reformat to how it was before):

```java
        return GodzillaUser.builder()
                .username(username)
                .password(passwordEncoder.encode("demo"))
                .authorities("ROLE_USER")
                .build();
```

**Review by instamine — APPROVED** [review]
Please comply to the coding conventions.

**Pull request merged**